api_spec/index.md

```diff
@@ -0,0 +1,9 @@
+# Node.js Client for Apache Ignite
+
+This thin client allows your Node.js applications to work with Ignite clusters via TCP.
+
+A thin client is a lightweight Ignite client that connects to the cluster via a standard socket connection. It does not start in JVM process (Java is not required at all), does not become a part of the cluster topology, never holds any data or used as a destination of compute grid calculations.
+
+What it does is it simply establishes a socket connection to a standard Ignite node and performs all operations through that node.
+
```

Comment:
Feels like the same thing is repeated 3 times here (TCP/socket connection/socket connection). A socket connection is how any DB driver works, do we need to put it here at all?

Reply:
You are right. Fixed.